.drone.yml

```diff
@@ -401,17 +434,6 @@ steps:
       cron: security_scans
       event: cron
 
-  - name: cron_anchore_scan
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
```

Comment:
We may want to keep the cron section, but do it with a replaced trivy scanning image